src/test/scala/com/cognite/sdk/scala/v1/AssetsTest.scala

```diff
@@ -61,7 +61,12 @@ class AssetsTest extends SdkTest with ReadBehaviours with WritableBehaviors {
     assetsToCreate,
     assetUpdates,
     (id: Long, item: Asset) => item.copy(id = id),
-    (a: Asset, b: Asset) => { a == b },
+    (a: Asset, b: Asset) => {
+      // Check all fields except lastUpdatedTime
+      a.externalId == b.externalId && a.name == b.name && a.id == b.id && a.aggregates == b.aggregates &&
```

Comment:
Easier way to do this is to `a.copy(lastUpdatedTime = 0) == b.copy(lastUpdatedTime = 0)`, then we don't need to change it when new fields are added.